resdk/resources/sample.py

```diff
@@ -21,6 +21,21 @@ def get_bam(self):
         """Return ``bam`` object on the sample."""
         return self.data.get(type='data:alignment:bam:')
 
+    def get_primary_bam(self, fallback_to_bam=False):
+        """Return ``primary bam`` object on the sample.
+
+        If the ``primary bam`` object is not present the function will
+        try to return a ``bam`` object if fallback_to_bam=True.
```

Comment:
Reword this to: 

    If the ``primary bam`` object is not present and ``fallback_to_bam`` is set
    to ``True``, a ``bam`` object will be returned.